must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -11,14 +14,14 @@ if [ "${BASE_COLLECTION_PATH}" = "" ]; then
 fi
 
 CEPH_COLLECTION_PATH="${BASE_COLLECTION_PATH}/ceph"
+# pre-install-ceph-resources.sh
+
+gather_external_ceph_resources ${BASE_COLLECTION_PATH} "${SINCE_TIME}"
```

Comment:
This file doesn't seem to exist...?

Reply:
yes, my bad,some files did not get committed, thanks